src/bookshelf/schema/sqitch.conf

```diff
@@ -0,0 +1,8 @@
+[core]
+	engine = pg
+	# plan_file = sqitch.plan
+	# top_dir = .
+# [engine "pg"]
+	# target = db:pg:
+	# registry = sqitch
+	# client = psql
```

Comment:
Are these lines all commented out? It seems like we actually do that across most of our `sqitch.conf` files. Any specific reason why?